apps/webapp/app/v3/marqs/sharedQueueConsumer.server.ts

```diff
@@ -566,6 +566,7 @@ export class SharedQueueConsumer {
                 lockedById: null,
                 status: lockedTaskRun.status,
                 startedAt: existingTaskRun.startedAt,
+                //should we set the completedAt to null here?
```

Comment:
@ericallam @nicktrn should we set `completedAt` to null here?

Reply:
Voting for `null`